docs/_includes/projects/docs/templates/databinding/my-element.js

```diff
@@ -3,10 +3,10 @@ import { LitElement, html } from 'lit-element';
 class MyElement extends LitElement {
   static get properties() {
     return {
-      prop1: String,
-      prop2: String,
-      prop3: Boolean,
-      prop4: String
+      prop1: {type: String},
```

Comment:
How about only including one `{type: string}` and make the others `{}` with a `// type: string is the default` comment?

Reply:
I think if we're going to show an empty declaration, it should probably be in the Properties chapter, not here.

I wasn't sure whether we should consider `{}` an anti-pattern or a convenient shorthand.

Reply:
`{}` in JS is the same as `@property()` in TS. It's perfectly fine.

Reply:
OK. I'm going to try to rationalize these samples in a subsequent PR. Just wanted to unbreak them here.